components/LandingPage/CreateButtons.tsx

```diff
@@ -1,13 +1,17 @@
+import { baseSepolia } from 'viem/chains'
 import CreateButton from '../CreateButton'
 import { useParams } from 'next/navigation'
 
 const CreateButtons = () => {
   const params = useParams()
   const collection = params.collection as string
+  const chainId = params.chainId
 
   return (
     <div className="flex justify-between w-full gap-2">
-      <CreateButton>{collection ? 'Create on Existing' : 'Preview'}</CreateButton>
+      <CreateButton chainId={collection ? Number(chainId) : baseSepolia.id}>
```

Comment:
keep fallback chainId consistent with shared `const`

```suggestion
      <CreateButton chainId={collection ? Number(chainId) : CHAIN_ID}>
```